examples/connext_dds/flat_data_api/c++98/CameraImage_subscriber.cxx

```diff
@@ -146,6 +146,11 @@ bool print_average_pixel_fast(const CameraImage &sample)
 
     unsigned int pixel_count = pixels.element_count();
 
+    if (pixel_count == 0) {
+        printf("The number of pixels should be greater than 0.");
```

Comment:
use `std::cout` instead
probably we can also change the message to "WARNING: the pixels count is 0"